src/electron/tools/router.ts

```
@@ -0,0 +1,188 @@
+
+## Example Response for "how many moons does Saturn have":
+**Saturn's Moons**
+
+Saturn has **146 confirmed moons**. This includes its largest moon Titan, which is larger than Mercury, and Enceladus, known for its subsurface ocean.
+
+<plan>
+Step 1: Provided information about Saturn's moon count based on current astronomical data
+</plan>
+`;
+
+const getRelevantContext = async (plan: string, context: any): Promise<string> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)